### ISSUE TITLE
discuss meeting Thursday

### ISSUE THREAD

**Issue by vyron-arvanitis** [scikit-learn-contrib/bde#55]
We noticed that, we build the memebrs inisde the fit_members method: 
https://github.com/vyron-arvanitis/bde/blob/b7a239460b26457bcf49655065efff53d1aab609/bde/bde_builder.py#L403-L404

that was done because,  I thought we want the user to give only the hidden layers, adn the first part we encounter data in the pipeline is the fit_members

We think this is wonrg, because fit memebrs shoudl fit memebrs and not create!->
maybe think another way or go back to the user creating the whole model structure

Do we want to have sth like regressor.history() -> and show the history of the training for all models?
model_1: training loss ....
model_2: training loss..

**Comment by EmanuelSommer** [comment]
I think we can close this one for now - thx for the fixes & optimizations
